### PR TITLE
Bump test dependencies

### PR DESCRIPTION
**Pull request by pavolloffay**
Signed-off-by: Pavol Loffay <ploffay@redhat.com>

### CODE REVIEW

File: opentracing-jaxrs2-itest/resteasy/src/test/java/io/opentracing/contrib/jaxrs2/itest/resteasy/RestEasyITest.java [hunk 1]
```diff
@@ -37,11 +44,19 @@ public void testAsyncError() {
      */
     @Test
     public void testAsyncErrorTestSpanReported() {
-        Client client = ClientBuilder.newClient();
-        Response response = client.target(url("/asyncError"))
-            .request()
-            .get();
-        response.close();
+        try {
```
Comment (jpkrohling) [inline opentracing-jaxrs2-itest/resteasy/src/test/java/io/opentracing/contrib/jaxrs2/itest/resteasy/RestEasyITest.java]: Minor, but you could use a try-with-resources here:

```java
try (CloseableHttpClient build = HttpClientBuilder.create().disableAutomaticRetries().build()) {
            Client client = new ResteasyClientBuilder().httpEngine(new ApacheHttpClient4Engine(build)).build();
            Response response = client.target(url("/asyncError"))
                .request()
                .get();
            response.close();
            response.readEntity(String.class); 

} catch (Exception ex) {
// client throws an exception if async request fails 
}
```

This way, your HttpClient still closes if you get an exception anywhere in the code. You can also do the same for the `response`, but that one is less "critical".

File: opentracing-jaxrs2-itest/resteasy/src/test/java/io/opentracing/contrib/jaxrs2/itest/resteasy/RestEasyITest.java [hunk 2]
```diff
@@ -44,18 +44,18 @@ public void testAsyncError() {
...
-                .get();
-            response.close();
+        // disable retry otherwise there can be 2 spans
+        CloseableHttpClient build = HttpClientBuilder.create().disableAutomaticRetries().build();
+        Client client = new ResteasyClientBuilder().httpEngine(new ApacheHttpClient4Engine(build)).build();
+        try (Response response = client.target(url("/asyncError"))
+            .request()
+            .get()) {
             response.readEntity(String.class);
             client.close();
```
Comment (jpkrohling) [inline opentracing-jaxrs2-itest/resteasy/src/test/java/io/opentracing/contrib/jaxrs2/itest/resteasy/RestEasyITest.java]: You shouldn't have the `client.close()` here, as it's in the finally block already

Comment (pavolloffay) [reply opentracing-jaxrs2-itest/resteasy/src/test/java/io/opentracing/contrib/jaxrs2/itest/resteasy/RestEasyITest.java]: oh my bad

Comment (pavolloffay) [reply opentracing-jaxrs2-itest/resteasy/src/test/java/io/opentracing/contrib/jaxrs2/itest/resteasy/RestEasyITest.java]: oh my bad

**Pull request merged**